tools/ci_build/github/azure-pipelines/templates/win-ci.yml

```diff
@@ -52,6 +57,11 @@ parameters:
   type: string
   default: 'onnxruntime'
 
+#for inference packages '', for training packages '-training'
+- name: extraArtifactNameSuffix
```

Comment:
nit: confusing to know how to use this when artifact_name_prefix has '-training in it'
> #for inference packages 'onnxruntime', for training packages 'onnxruntime-training'

and extraArtifactNameSuffix says pass '-training' in that as well
> #for inference packages '', for training packages '-training'

can we have one arg for the 'onnxruntime' and one for any optional suffix like '-training' and put together the right names with just that?

e.g. base_artifact_name = prefix + suffix, and apply the suffix to other things as needed?

Reply:
Done!